include/flamegpu/runtime/DeviceAPI.cuh

```diff
@@ -88,6 +88,14 @@ class ReadOnlyDeviceAPI {
         return getVariable<id_t>("_id");
     }
 
+    /**
+     * Access the current stepCount
+     * @return the current step count, 0 indexed unsigned.
+     */
+    __forceinline__ __device__ unsigned int getStepCounter() const {
```

Comment:
~would `getCurrentStep` be a better name? (just a suggestion while viewing the diff)~

Just spotted this was a move not an addition, so probably better to avoid the breaking change.